Resources/Prototypes/DeltaV/Entities/Objects/Devices/station_beacon.yml

```
@@ -1,113 +1,113 @@
 #Command
```

yamllint

[warning] missing starting space in comment

(comments)